main.go

```diff
@@ -70,6 +71,16 @@ func mainErr() error {
 		if ctx.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+
+		path := cmd.GetConfigPath(ctx)
+		warnings, err := config.GetFilePermissionWarnings(path)
+		if err != nil {
+			return err
```

Comment:
I don't think that it make sense to return an error in this case. I'd be concerned that a failure here would prevent the rest of the application from running, and since we don't use this call for anything except warning, I don't think that this should be a blocking failure.

Reply:
What would make more sense between ignoring the error completely vs logging the error and continuing?

Reply:
Agree with your choice, I think logging is the most consistent choice.